opus/application/static_media/js/browse.js

```diff
@@ -575,7 +578,19 @@ var o_browse = {
             }
         });
 
+        // make sure slider won't scroll with keyboard
+        $(".op-slider-pointer").unbind("keydown");
+
         $(document).on("keydown click", function(e) {
+            // Whenever we click or hit up/down arrows, page up and page down keys, we
+            // want to make sure the ps container is hovered and ps is focused so the the
+            // keydown event will be properly handled by ps.
+            if (opus.prefs.view === "browse" && (e.type === "click" ||
+                (e.which || e.keyCode) == 38 || (e.which || e.keyCode) == 40 ||
+                (e.which || e.keyCode) == 33 || (e.which || e.keyCode) == 34)) {
+                o_browse.hoverAndFocusOnPS();
+            }
+
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/SETI/rms-opus/pull/1387#issuecomment-2276954910>#c06976</a><span> - 08/09/2024, 12:49 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The use of 'e.which' and 'e.keyCode' is deprecated and may not be supported in future versions of browsers. This can lead to potential issues with cross-browser compatibility and future-proofing the code.
</div>
<div id="fix">
<b>Fix:</b> Replace the deprecated 'e.which' and 'e.keyCode' with 'e.key' or 'e.code' to ensure future compatibility and adherence to modern standards.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -588,6 +588,6 @@
              if (opus.prefs.view === "browse" && (e.type === "click" ||
 -                (e.which || e.keyCode) == 38 || (e.which || e.keyCode) == 40 ||
 -                (e.which || e.keyCode) == 33 || (e.which || e.keyCode) == 34)) {
 +                e.key === "ArrowUp" || e.key === "ArrowDown" ||
 +                e.key === "PageUp" || e.key === "PageDown")) {
                  o_browse.hoverAndFocusOnPS();
              }
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged